cinema/urls.py

```diff
@@ -1 +1,28 @@
-# write urls here
+from django.urls import path, include
+from cinema.views import (
+    GenreViewSet,
+    ActorViewSet,
+    CinemaHallViewSet,
+    MovieViewSet,
+    MovieSessionViewSet,
+)
+from rest_framework import routers
+
+
+app_name = "cinema"
+
+
+router = routers.DefaultRouter()
+router.register("genres", GenreViewSet, basename="genre")
+router.register("actors", ActorViewSet, basename="actor")
+router.register("cinema_halls", CinemaHallViewSet, basename="cinema_hall")
+router.register("movies", MovieViewSet, basename="movie")
+router.register(
+    "movie_sessions",
+    MovieSessionViewSet,
+    basename="movie_session"
+)
+
+urlpatterns = [
+    path("", include(router.urls)),
+]
```

Comment:
If your path equal to empty string, no reason to specify it if router has been created. Use `urlpatterns = router.urls` instead